src/claim/authorize-ksign/authorize-ksign.js

```diff
@@ -0,0 +1,72 @@
+const Claim = require('../claim');
+const utils = require('../../utils');
+
+/**
+ * @param  {Uint32} Version
+ * @param  {Bool} Signe
+ * @param  {String} Ax
+ * @param  {String} Ay
+ */
+class AuthorizeKSign {
+  constructor(_version = 0, _signe = false, _ax = '', _ay = '') {
+    const versionBuff = Buffer.alloc(4);
```

Comment:
We should have in the comments why are you allocating 4 bytes, 1 bytes, etc... 
Or the claim structure commented ....